apps/modernization-ui/src/apps/page-builder/page/management/edit/question/QuestionHeader.tsx

```diff
@@ -20,13 +23,16 @@ const lineSeparatorId = 1012;
 const originalElecDocId = 1036;
 const readOnlyPartId = 1030;
 
+const staticTypes = [hyperlinkId, commentsReadOnlyId, lineSeparatorId, originalElecDocId, readOnlyPartId];
+
 export const QuestionHeader = ({
     question,
     onRequiredChange,
     onEditQuestion,
     onDeleteQuestion,
     visible = true
 }: Props) => {
+    const editStaticElementRef = useRef<ModalRef>(null);
```

Comment:
Issue: creating the `ModalRef` here will create a new modal ref for each question on the page. I highly recommended moving this up so that no more than 1 `ModalRef` is created for the page

Reply:
+1